src/domain/Lists/DraggableList/DraggableList.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import {
+  DragDropContext,
+  Draggable,
+  DropResult,
+  Droppable
+} from 'react-beautiful-dnd'
+import uuid from 'uuid'
+
+import { Props } from '../../../common'
+import { StyledListArea, StyledListItem } from './style'
+
+interface IDraggableListProps<T> {
+  /** The array of items */
+  items: T[]
```

Comment:
So Tandrew talked to me about removing this key because it isn't actually needed. React setState/useState setters have more functionality than just setting the state to the value you pass in. 

If you pass in a callback instead you get a reference to the existing state. eg:
```
[simpsons, setSimpsons] = useState(['Homer', 'Bart'])

console.log(simpsons)  // First render: ['Homer', 'Bart'])
                       // Second render: ['Marge', 'Homer', 'Bart', 'Lisa', 'Maggie']

setSimpsons((existingSimpsons) => {
  console.log(existingSimpsons) // ['Homer', 'Bart'])
  return [
   'Marge'
    ...existingSimpsons,
   'Lisa',
   'Maggie'
  ]
})
```